creator/Documents/CommandBlocks.md

```diff
@@ -25,127 +25,116 @@ In this guide, we will set up a command block chain to look for a certain type o
-1. Close the GUI.
+1. In the Command Input section, input the following: `/testforblock 0 4 0 diamond_block`.
+1. Close the interface.
 
 ## Command Block Chains
 
-Command blocks have multiple types it can be set to. A command block set to a Chain type means that it will execute its' command if the previous command block executed. You can connect multiple types of command blocks and chain them together to execute a series of logic. The next step in the gameplay loop is to check if the player is able to receive a reward.
+For the next step, we need to place a chain command block. As mentioned above, chain command blocks execute when a block pointing into them is activated. Make sure you place the block like the image showing the arrows.
 
 1. Place another command block next to the existing block. Make sure that the new block is placed in front of the direction that the arrow is pointing.
-1. Click the **Use** button on the newly created block to open the command block GUI.
+1. Open the command block interface.
 1. Set the **Block Type** to **Chain**.
 1. Set the **Condition** to **Conditional**.
 1. Set the **Redstone** to **Always Active**.
-1. Set the **Command Input** to  `/scoreboard players test @p wool_placed 0 0`
+1. Set the **Command Input** to  `/testfor @p[tag=!placed_block]`
 
-Since the second block is set to **Conditional**, the block will only execute if the first command block gives a true value. If a player places a wool block in the correct location, the next block will check if the player's Wool_placed score is 0.
+Since the second block is set to **Conditional**, this new block will only execute if the first command block succeeds, meaning the diamond block was present. This command checks to see if the player hasn't already received the reward.
 
 Let’s add a third block to set a command to reward the player for placing a block in the right spot.
 
 1. Place another command block next to the existing block. Make sure that the new block is placed in front of the direction that the arrow is pointing.
-1. Click the **Use** button on the newly created block to open the command block GUI.
+1. Open the command block interface.
 1. Set the **Block Type** to **Chain**.
 1. Set the **Condition** to **Conditional**.
 1. Set the **Redstone** to **Always Active**.
 1. Set the **Command Input** to `/give @p emerald`.
 
-Now that the player has received the emerald, the last step is to then close the loop by giving the player a score value of 1. This will then set the 2nd block in the chain to false.
+Once again, this command will only activate if the previous command succeeded, meaning both the diamond block was present and the player hasn't received the reward yet.
+
+If we stop now, placing the diamond block at `0 4 0` will give the player 20 emeralds per second! That's because we need to do one last step, marking the player as having received the reward.
```

Comment:
I changed this because this is a good opportunity to temporarily step back from "do this because I said so" and let the learner think critically about the steps they've done so far. It's brief, but a nice reprieve from the many steps we've done so far, and if they try it, they get to see something interesting.